frontend/src/elements/AppointmentGridItem.vue

```diff
@@ -7,7 +40,7 @@
     }"
     :style="{ borderColor: appointment.calendar_color }"
     @mouseover="el => !isPast ? paintBackground(el, appointment.calendar_color, '22') : null"
-    @mouseout="el => !isPast ? paintBackground(el, appointment.calendar_color, _, true) : null"
+    @mouseout="el => !isPast ? paintBackground(el, appointment.calendar_color, '', true) : null"
```

Comment:
Idk how to skip typed parameters, so I gave a (irrelevant) default argument here.

Reply:
undefined might work? That's the value if a parameter is skipped by default in js at least.

Reply:
Perfect, thanks